package.json

```diff
@@ -98,7 +98,7 @@
   "scripts": {
     "enforce": "enforce-node-version",
     "preinstall": "node scripts/preinstall.js",
-    "postinstall": "yarn run enforce && node scripts/yarn/apps-install",
+    "postinstall": "yarn run enforce && node scripts/yarn/apps-install && ./scripts/check-npm-cache.sh",
```

Comment:
Should we be consistent with the paths here? So one of these:

`./scripts/yarn/apps-install && ./scripts/check-npm-cache.sh`
`./scripts/yarn/apps-install/index.sh && ./scripts/check-npm-cache.sh`